middle_finger:
        dt: 0.01
        M_joint:
                kp: 0.2
                ki: 0.001
                kd: 0.001
        P_joint:
                kp: 0.2
                ki: 0.05
                kd: 0.0005
        D_joint:
                kp: 0.1
                ki: 0
                kd: 0.0001
                
